File: Uri-Iniciante/1013.java
import java.io.IOException;
import java.util.Scanner;
/*
 * 
*/

public class Main {
	public static int abs(int x) {
		if(x<0) {
			return -x;
		}else
			return x;
	}
	public static void main(String[] args) throws IOException {
		Scanner sc = new Scanner(System.in);
		int A,B,C,maiorAB;
		A = sc.nextInt();
		B = sc.nextInt();
		C = sc.nextInt();
		
		maiorAB = ((A+B)+abs(A-B))/2;
		
		if(maiorAB>C) {
			System.out.println(maiorAB+" eh o maior");
		}else
			System.out.println(C+" eh o maior");
		
		
		sc.close();
	}

}

File: Uri-Iniciante/1013.cpp
#include<stdio.h>
/*	
*	-----------------------------------
*	|		Pedro Daniel Jardim       |
*	| 			  UFV				  | 
*	|		  25/02/2020			  |
*   ----------------------------------
*
*/
int abs(int x) {
		if(x<0) {
			return -x;
		}else
			return x;
	}

int main() 
{
		int A,B,C,maiorAB;
		scanf("%d %d %d",&A,&B,&C);
		
		maiorAB = ((A+B)+abs(A-B))/2;
		
		if(maiorAB>C) {
			printf("%d eh o maior \n",maiorAB);
		}else
			printf("%d eh o maior \n",C);
		
 
   return 0;
   
}
